File: dsa_python/bubble_sort.py
def bubble_sort(arr):
    n = len(arr)
    for i in range(n-1):
        for j in range(n-i-1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]
    return arr

my_array = [64, 34, 25, 12, 22, 11, 90, 5]
print('Actual array is:', my_array)
print('Sorted array is:', bubble_sort(my_array))
print('Sorted array is:', my_array)

File: dsa_js/bubble_sort.js
function bubble_sort(arr) {
    const len = arr.length;
    for (let i = 0; i < len; i++) {
        for (let j = 0; j < len - i; j++) {
            if (arr[j] > arr[j + 1]) {
                let temp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = temp;
            }
        }
    }
    return arr;
}

const list = [7, 12, 9, 4, 11];
console.log('Actual array:', list);
console.log('Sorted array:', bubble_sort(list));
console.log('Sorted array:', list);